src/hotspot/share/gc/shenandoah/shenandoahFreeSet.hpp

```diff
@@ -29,30 +29,136 @@
 #include "gc/shenandoah/shenandoahHeapRegionSet.hpp"
 #include "gc/shenandoah/shenandoahHeap.hpp"
 
-class ShenandoahFreeSet : public CHeapObj<mtGC> {
+enum FreeMemoryType : uint8_t {
```

Comment:
Globally visible type should `Shenandoah` prefix.

Reply:
Thanks.  Done.